transformations/unit_converter/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "unit_converter",
+  "test_cases": [
+    {
+      "class": "UnitConverter",
+      "inputs": {
+        "sentence": "I weigth 100 pounds."
```

Comment:
`weigth` to `weigh`?

Reply:
Thank you for spotting the typo. I have fixed it in 7238ec1a971339afb564bc4eb4e32f82eec684f2